packages/progress/progress.vue

```diff
@@ -0,0 +1,76 @@
+<template lang="pug">
+div.zi-progress-bar(:class="fix")
+  div.zi-progress
+    div.zi-progress__inner(:style="styles")
+  span.zi-progress__text {{ percentage <= 100 ? percentage : 100 }}
+</template>
+
+<script>
+import { validator } from '../utils'
+
+export default {
+  name: 'zi-progress',
+
+  props: {
+    percentage: {
+      type: Number,
```

Comment:
```suggestion
      type: [String, Number],
```
```js
Number.isNaN(Number(percentage)) && throw new Error('....')
```
兼容 String，转成 number，如果不能转成 number 再提示错误。